code/modules/surgery/organs/autoimplanter.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/device/autoimplanter
+	name = "autoimplanter"
+	desc = "A device that automatically injects a cyber-implant into the user without the hassle of extensive surgery. It has a slot to insert implants and a screwdriver slot for removing accidentally added implants."
+	icon_state = "gangtool-white"
+	item_state = "walkietalkie"
```

Comment:
this doesn't even have an in-hand anywhere; pretty sure this item_state doesn't exist.